src/index.js

```diff
@@ -1,47 +1,63 @@
 import './style.css';
-
 import { isCompleted, isDone } from './status.js';
+import {
+  add, edit, remove, removeChecked,
+} from './addremove.js';
 
 const storage = window.localStorage;
-
-const task = [
-  {
-    description: 'Adding a new item',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Going shop',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Do Exercise',
-    completed: false,
-    index: 2,
-  },
-];
+const task = [];
+const addBtn = document.getElementById('addBtn');
+const clear = document.getElementById('clear');
+const form = document.getElementById('form');
 
 function render(taskStorage) {
   taskStorage.forEach((tsk) => {
     const { description } = tsk;
     const list = document.getElementById('list');
     const item = document.createElement('li');
-    const checkBox = document.createElement('input');
-    checkBox.setAttribute('type', 'checkbox');
-    checkBox.checked = isDone(tsk);
+    const checkbox = document.createElement('input');
     const itemText = document.createElement('p');
-    const threeDot = document.createElement('i');
-    threeDot.className = 'fas fa-ellipsis-v';
+    // const threeDot = document.createElement('i');
+    const removeIcon = document.createElement('i');
+
+    checkbox.setAttribute('type', 'checkbox');
+    checkbox.id = 'checkbox';
+    checkbox.checked = isDone(task);
+    // threeDot.className = 'fas fa-ellipsis-v';
+    removeIcon.className = 'far fa-trash-alt';
+    removeIcon.id = 'removebtn';
     itemText.innerHTML = description;
+
     list.appendChild(item);
-    item.appendChild(checkBox);
+    item.appendChild(checkbox);
     item.appendChild(itemText);
-    item.appendChild(threeDot);
-    checkBox.addEventListener('click', () => {
-      isCompleted(checkBox.checked, tsk);
+    // item.appendChild(threeDot);
+    item.appendChild(removeIcon);
+
+    removeIcon.addEventListener('click', () => {
+      remove(taskStorage, task);
+      window.location.reload();
+    });
+
+    checkbox.addEventListener('click', () => {
+      itemText.style.textDecoration = 'line-through';
+      checkbox.addEventListener('click', () => {
+        itemText.style.textDecoration = 'none';
+      });
+      isCompleted(checkbox.checked, tsk);
       storage.setItem('stored', JSON.stringify(taskStorage));
     });
+
+    itemText.addEventListener('click', () => {
+      itemText.setAttribute('contenteditable', 'true');
+    });
+
+    itemText.addEventListener('keypress', (e) => {
+      if (e.key === 'Enter') {
+        edit(task, itemText.innerText);
+        window.location.reload();
+      }
```

Comment:
- [ ] Kindly change all regular functions to es6 arrow function syntax following the JavaScript best practices guide.